assets/schema_input.json

```diff
@@ -8,13 +8,12 @@
         "type": "object",
         "properties": {
             "sample": {
-                "type": "string",
+                "type": ["string", "integer"],
                 "errorMessage": "Sample name must be provided and cannot contain spaces",
                 "meta": ["id"]
             },
             "run_accession": {
-                "type": "string",
-                "unique": ["sample"],
+                "type": ["string", "integer"],
                 "errorMessage": "Run accession must be provided and cannot contain spaces."
```

Comment:
Same here

Reply:
`regex` corresponding to this errorMessage should e added